src/Symfony/Bundle/WebServerBundle/WebServer.php

```diff
@@ -150,7 +150,9 @@ private function createServerProcess(WebServerConfig $config)
             throw new \RuntimeException('Unable to find the PHP binary.');
         }
 
-        $process = new Process(array_merge(array($binary), $finder->findArguments(), array('-dvariables_order=EGPCS', '-S', $config->getAddress(), $config->getRouter())));
+        $xdebugArgs = \extension_loaded('xdebug') ? array('-dxdebug.profiler_enable_trigger=1') : array();
```

Comment:
should we check whether this setting is enabled in the current PHP instead of always adding them ? AFAIK, turning on this feature adds overhead to the execution of PHP, as Xdebug needs to instrument the code (even though it does not actually collect the profiling data).

Reply:
It's the reason why I turn on only the trigger of the profiler but not the profiler itself, whose the overhead is insignifiant.